cfy_manager/components/cluster/cluster.py

```diff
@@ -91,111 +92,85 @@ def _generic_cloudify_rest_request(self, host, port, path,
             raise BootstrapError(
                 'REST service returned malformed JSON: {0}'.format(e))
 
-    def _get_join_addresses(self, master_manager_ip):
+    def _get_current_version(self):
         result = self._generic_cloudify_rest_request(
-            master_manager_ip,
+            config[MANAGER][PRIVATE_IP],
             80,
-            'v3.1/cluster/nodes',
-            'get')
-        return [n['host_ip'] for n in result['items']]
+            'v3.1/version',
+            'get'
+        )
+        return result
 
-    def _add_new_cluster_node_on_master(self, master_manager_ip,
-                                        cluster_node_host_ip,
-                                        cluster_node_name,
-                                        cluster_node_database_ip):
+    def _join_to_cluster(self):
+        """
+        Used for either adding the first node to the cluster (can be
+        single-node cluster), or adding a new manager to the cluster
+
+        The next step would be installing the mgmtworker which requires the
+        rest-security.conf to be the same as the rest of the cluster, as a
```

Comment:
"as the rest" => "as in the rest"
or maybe "as in the other managers in the cluster"

Reply:
Fixed